db/20200521145827_add_is_archived_to_cron_wfs_and_wf_execs.sql

```diff
@@ -0,0 +1,8 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+ALTER TABLE cron_workflows ADD COLUMN is_archived BOOL;
+ALTER TABLE workflow_executions ADD COLUMN is_archived BOOL;
```

Comment:
@aleksandrmelnikov Let's add a `DEFAULT false;` to these columns. I forget if that will make existing records have `false` for the value. If they don't, we need to run an update query so existing ones are marked as archived false.

@rushtehrani Should these columns be nullable? I'm leaning towards no, though it might be redundant with a DEFAULT set.

Reply:
They should not be nullable. Only acceptable values are `true` or `false` in this case.

Reply:
@rushtehrani Makes sense. Okay, we'll have to fix the other tables too, I believe they are nullable (workflow and workspace templates).

Sample query for updating an existing column:
```sql
ALTER TABLE workflow_templates ADD is_archived BOOL;
UPDATE workflow_templates SET is_archived = false;
ALTER TABLE workflow_templates ALTER COLUMN is_archived SET NOT NULL;
```

Reply:
Refactored to this:
```
ALTER TABLE cron_workflows ADD COLUMN is_archived BOOL DEFAULT false NOT NULL;
ALTER TABLE workflow_executions ADD COLUMN is_archived BOOL DEFAULT false NOT NULL;
```
This creates default false, and makes it not null